src/MocosSimLauncher.jl

```diff
@@ -93,14 +93,16 @@ function launch(args::AbstractVector{T} where T<:AbstractString)
   else
     error("the import function was not used!")
   end
-
   @threads for trajectory_id in 1:num_trajectories
     state = states[threadid()]
     MocosSim.reset!(state, trajectory_id)
     
     for fun in import_funcs
       fun(state, params)
     end
+    if params.screening_params != nothing
```

Comment:
compare to nothing with `!==` and `===`or with `isnothing`
```suggestion
    if params.screening_params !== nothing
```